mmdeploy/backend/coreml/torchscript2coreml.py

```diff
@@ -52,9 +52,10 @@ def from_torchscript(torchscript_model: Union[str,
                      output_file_prefix: str,
                      input_names: Sequence[str],
                      output_names: Sequence[str],
-                     input_shapes: Dict,
+                     input_shapes: Dict[str, Dict],
+                     compute_precision: str = 'FLOAT32',
                      convert_to: str = 'neuralnetwork',
-                     fp16_mode: bool = False,
+                     minimum_deployment_target: Optional[str] = None,
```

Comment:
What value should be passed to `minimum_deployment_target`?

Reply:
According to the [config file](https://github.com/open-mmlab/mmdeploy/blob/master/configs/_base_/backends/coreml.py) I guess it is a string that describes the OS or device. Such as 'IOS16'

Reply:
Let's clarify it more clear in docstring